AdyenCheckout/CheckoutComponentBuilder.swift

```diff
@@ -15,14 +15,12 @@
     @_spi(AdyenInternal) import AdyenActions
 #endif
 
-public protocol AdyenCheckoutComponent: PresentableComponent {}
-
 internal enum CheckoutComponentBuilder {
     
     internal static func build(
         for paymentMethod: PaymentMethod,
         configuration: CheckoutConfiguration
-    ) throws -> AdyenCheckoutComponent {
+    ) -> PaymentComponent {
```

Comment:
We need to keep `throws` for sake of ApplePay and other SDK-based payment components that might not be generated directly.

Reply:
It is removed for now for simplicity. Whenever needed, we will add it back.